Automattic/pocket-casts-android
# End of Year: Epilogue story design

| 📘 Project: #410 |
|:---:|

iOS PR: https://github.com/Automattic/pocket-casts-ios/pull/467

## Description

- Adds the design for the epilogue story.
- Improves pause/ resume behavior (now the story pause on press instead of long press earlier)

## Testing Instructions
1. Set `END_OF_YEAR_ENABLED` feature flag to `true` in base.gradle
2. Run the app and make sure you're logged in to an account with a few items on Listening History
3. When the prompt appears, tap "View My 2022"
4. Navigate to the last story
5. ✅ Check the design
6. Tap "Replay"
7. ✅ Stories should replay
8. Tap on a story
9.  ✅ Story should pause immediately
10. Release tap event
11.   ✅ Story should resume



## Screenshots or Screencast 
<img width=320 src="https://user-images.githubusercontent.com/1405144/201554551-51e23bbb-3521-4318-ab8f-5c7d6b68dd32.png"/>

P.S. I picked the background color from the iOS app and applied the existing gradient on other screens.

## Checklist
- If this is a user-facing change, I have added an entry in CHANGELOG.md
- I have considered whether it makes sense to add tests for my changes
- Any jetpack compose components I added or changed are covered by compose previews
 
#### I have tested any UI changes...
